config/notifiers.go

```diff
@@ -456,16 +456,17 @@ type OpsGenieConfig struct {
 
 	HTTPConfig *commoncfg.HTTPClientConfig `yaml:"http_config,omitempty" json:"http_config,omitempty"`
 
-	APIKey      Secret                    `yaml:"api_key,omitempty" json:"api_key,omitempty"`
-	APIURL      *URL                      `yaml:"api_url,omitempty" json:"api_url,omitempty"`
-	Message     string                    `yaml:"message,omitempty" json:"message,omitempty"`
-	Description string                    `yaml:"description,omitempty" json:"description,omitempty"`
-	Source      string                    `yaml:"source,omitempty" json:"source,omitempty"`
-	Details     map[string]string         `yaml:"details,omitempty" json:"details,omitempty"`
-	Responders  []OpsGenieConfigResponder `yaml:"responders,omitempty" json:"responders,omitempty"`
-	Tags        string                    `yaml:"tags,omitempty" json:"tags,omitempty"`
-	Note        string                    `yaml:"note,omitempty" json:"note,omitempty"`
-	Priority    string                    `yaml:"priority,omitempty" json:"priority,omitempty"`
+	APIKey                 Secret                    `yaml:"api_key,omitempty" json:"api_key,omitempty"`
+	APIURL                 *URL                      `yaml:"api_url,omitempty" json:"api_url,omitempty"`
+	Message                string                    `yaml:"message,omitempty" json:"message,omitempty"`
+	Description            string                    `yaml:"description,omitempty" json:"description,omitempty"`
+	Source                 string                    `yaml:"source,omitempty" json:"source,omitempty"`
+	Details                map[string]string         `yaml:"details,omitempty" json:"details,omitempty"`
+	DetailsUseCommonLabels bool                      `yaml:"details_use_common_labels,omitempty" json:"details_use_common_labels,omitempty"`
```

Comment:
I would propose:

```suggestion
	CommonLabelsAsDetails bool                      `yaml:"common_labels_as_details,omitempty" json:"common_labels_as_details,omitempty"`
```
as per my other comment.